Makefile

```diff
@@ -171,6 +171,9 @@ pkg/mock/mock_scaling/mock_executor/mock_interface.go: pkg/scaling/executor/scal
 	$(MOCKGEN) -destination=$@ -package=mock_executor -source=$^
 pkg/mock/mock_scaler/mock_scaler.go: pkg/scalers/scaler.go
 	$(MOCKGEN) -destination=$@ -package=mock_scalers -source=$^
+pkg/mock/mock_secretLister/mock_interfaces.go: vendor/k8s.io/client-go/listers/core/v1/secret.go
```

Comment:
@Indresh2410 I guess that `pkg/mock/mock_secretlister/mock_interfaces.go` needs to be added to `mockgen-gen` target above. @zroubalik could you please confirm? Thanks

Reply:
@tobotg good catch, you are right!